utils/criu_linux.go

```diff
@@ -0,0 +1,45 @@
+//go:build linux
+// +build linux
```

Comment:
Since the filename is suffixed with `_linux`, it will be included only in Linux builds. Do you think we need to keep the build constraints within the file too? It seems redundant to me.

Reply:
Ah, good question. Just recently I had a problem with something like that. Let me try it again.

Reply:
Yes, that would still need the constraints, but we can omit it just for this file.